router/apiHandler.go

```diff
@@ -173,15 +179,21 @@ func ApiUploadHandler(w http.ResponseWriter, r *http.Request) {
 }
 
 func ApiUpdateHandler(w http.ResponseWriter, r *http.Request) {
-	if config.UploadsDisabled {
+	user := model.User{}
```

Comment:
But where is it read in the old file? I mean I just moved it up a few lines and from my understanding nothing redecleared it to fill it